# Задание 1

Известно, что генеральная совокупность распределена нормально
со средним квадратическим отклонением, равным 16.
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
если выборочная средняя M = 80, а объем выборки n = 256.

a = 0.05  
Z = 1.96   
80 +- 1.96 * 16 / 256**0.5 = 80 +- 1.96, т.е. 78.04 < m < 81.96

# Задание 2

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,
получены опытные данные:   
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1   
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
значение с доверительной вероятностью 0,95.

In [1]:
import numpy as np
x = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
x.std(ddof=1)

0.4508017549014448

In [2]:
x.mean()

6.590000000000001

a = 0.05   
t(a/2) = +-2.262   
n = 10   
std = 0.45   
m = 6.59 

In [3]:
6.59 + 2.262*0.45 / 10**0.5

6.911888243028539

In [4]:
6.59 - 2.262*0.45 / 10**0.5

6.268111756971461

[6.268; 6.912]

# Задание 3

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм.

mu0 = 17   
mu1 = 17.5   
H1: mu1 > mu0   
s = 1.645   
std = 2   
SE = 0.2   
(17.5 - 17) / 0.2 = 0.5 / 0.2 = 2.5   
2.5 > s, значит, гипотеза H1 верна, и средний диаметр не 17 мм.


# Задание 4

Продавец утверждает, что средний вес пачки печенья составляет 200 г.
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%? (использовать двусторонний тест)

In [5]:
y = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
y.std(ddof=1)

4.453463071962462

In [6]:
y.mean()

198.5

mu0 = 200   
H1: mu1 =/ mu0
a = 0.01  
t(a/2) = +-3.25
n = 10    
std = 4.45   
mu1 = 198.5   

In [7]:
T = (198.5 - 200) / (4.45 / (10)**0.5)
T

-1.0659362899443976

Соответственно, при а = 0.01 верна гипотеза Н0